.NET/Microsoft.Recognizers.Definitions/Japanese/ChoiceDefinitions.cs

```diff
@@ -0,0 +1,25 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+//     
+//     Generation parameters:
+//     - DataFilename: Patterns\Japanese\Japanese-Choice.yaml
+//     - Language: Japanese
+//     - ClassName: OptionsDefinitions
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Microsoft.Recognizers.Definitions.Japanese
+{
+	using System;
+	using System.Collections.Generic;
+
+	public static class OptionsDefinitions
+	{
+		public const string LangMarker = "Jp";
```

Comment:
Please change the LangMarker in the YAML file to "Jpn" and re-generate this file.